servers/src/main/java/tachyon/master/block/meta/MasterWorkerInfo.java

```diff
@@ -271,6 +271,15 @@ public synchronized void updateLastUpdatedTimeMs() {
   }
 
   /**
+   * Updates the last updated time of the worker in ms. Should only be called by tests.
+   *
+   * @param timeMillis the time to set as the last updated time, in ms.
+   */
+  public synchronized void updateLastUpdatedTimeMs(long timeMillis) {
```

Comment:
In general for setters, the pattern should be:

```
public void setFoo(T foo) {
  mFoo = foo;
}
```
in this case that would mean:
```
public synchronized void setLastUpdatedTimeMs(long lastUpdatedTimeMs) {
  mLastUpdatedTimeMs = lastUpdatedTimeMs;
}
```